index.js

```diff
@@ -0,0 +1,217 @@
+//. # Sanctuary Benchmark
+//.
+//. Small wrapper over [benchmarkjs][1] to enable a consistent style of
+//. benchmarking across all Sanctuary projects. Allows for the easy definition
+//. of comparative benchmarks, and outputs results in a standardized format for
+//. sharing.
+//.
+//. ## Usage
+//.
+//. ```js
+//. //
+//. // old-vs-new.js
+//. //
+//.
+//. //Imagine these are libs. Normally they would be require()'d.
+//. const oldVersion = (f, xs) => xs.map(f);
+//. const newVersion = (f, xs) => {
+//.   const len = xs.length; out = new Array(len);
+//.   for(let idx = 0; idx < len; idx++) out[idx] = f(xs[idx]);
```

Comment:
Nit: Please add a space after `for`. Also, I'd slightly prefer `idx += 1` or `++idx` to `idx++`.